evap/settings.py

```diff
@@ -402,6 +404,14 @@ def CHARACTER_ALLOWED_IN_NAME(character):  # pylint: disable=invalid-name
 OIDC_OP_JWKS_ENDPOINT = "https://example.com/certs"
 
 
+### Content Security Policy
+CSP_OBJECT_SRC = ("'none'", )
+CSP_SCRIPT_SRC = ("'strict-dynamic'", )  # scripts require a correct nonce or to be loaded by a trusted script
+CSP_STYLE_SRC = ("'unsafe-inline'", "'self'") # inline-styles are allowed
+CSP_IMG_SRC = ("'self'", "data:") # images from same host or as data: url are allowed
+CSP_BASE_URI = ("'none'", )
+CSP_INCLUDE_NONCE_IN=['script-src']
+
```

Comment:
I'd guess getting notified about CSP errors in production would be more beneficial than in tests. We may circumvent the spam problem by restricting the report endpoint to logged-in users. I don't yet know how the reporting mechanism works though, and whether it includes the current session (and thus allows to require a logged in user). Will look into it.

Setting up an endpoint just to use it in tests feels overengineered to me, and I also don't really know if/how we could use it. I guess it would have to be an endpoint that simply raises an error to fail the test?

Reply:
Just wanted to hear whether you already had an opinion on the topic. I agree with what you wrote, let's pick this up in a separate PR

Reply:
Added as a discussion point to the OP